pkgs/development/tools/rust/cargo-msrv/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, nix-update-script
+, makeWrapper
+, pkg-config
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-msrv";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "foresterre";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1ynv5d2rxlc1gzq93v8qjyl5063w7q42g9m95250yh2lmf9hdj5i";
+  };
+
+  cargoSha256 = "03rphdps17gzcmf8n5w14x5i5rjnfznsl150s3cz5vzhbmnlpszf";
+
+  passthru = {
+    updateScript = nix-update-script {
+      attrPath = pname;
+    };
+  };
+
+  doCheck = false;
+
+  buildInputs = [ openssl ];
+
+  nativeBuildInputs = [ makeWrapper pkg-config ];
```

```suggestion
  nativeBuildInputs = [ pkg-config ];
```
I'm not sure if you need `makeWrapper` here. On my system it also works without.

---

Fixed.